### ISSUE TITLE
Placeholder in prompt for existing rem content 

### ISSUE THREAD

**Issue by ghost** [bjsi/prompt-explorer#7]
Creating custom prompts is great. It could be even more powerful if the prompt text could contain a placeholder for an existing (selected) rem's content. For example:

_Rephrase the following text so a third-grader can understand it: **[rem]**_

This way, the prompts can relate to any selected rem by using the **[rem]** placeholder

**Comment by bjsi** [comment]
agreed, i actually implemented this in another branch, i will get round to
it eventually!

**Comment by ev01ve** [comment]
Came here looking for exactly this.  Can't wait to try it!
